src/icons/Check/Check.web.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { getThemeColors } from '../../_helpers/theme';
+import colors from '../../tokens/colors';
+
+function Check({ width, height, fill }) {
+  return (
+    <svg
```

Comment:
Add `Check.native.js` as well.
Look other Icons. You would just have to use `Svg` and `Path` components.

Reply:
added